twitter.html

```diff
@@ -0,0 +1,568 @@
+                        viewBox="0 0 122.88 122.88" style="enable-background:new 0 0 122.88 122.88"
+                        xml:space="preserve">
+                        <g>
+                            <path
+                                d="M0,61.44C0,44.48,6.88,29.12,18,18C29.11,6.88,44.47,0,61.44,0c16.96,0,32.33,6.88,43.44,18c11.12,11.12,18,26.48,18,43.44 s-6.88,32.33-18,43.44c-11.12,11.12-26.48,18-43.44,18c-16.96,0-32.33-6.88-43.44-18C6.88,93.77,0,78.41,0,61.44L0,61.44z M92.3,61.44c0,3.99-3.24,7.22-7.22,7.22s-7.22-3.24-7.22-7.22c0-3.99,3.23-7.23,7.22-7.23S92.3,57.45,92.3,61.44L92.3,61.44 L92.3,61.44z M43.99,61.44c0,3.99-3.23,7.22-7.23,7.22c-3.99,0-7.22-3.24-7.22-7.22c0-3.99,3.23-7.23,7.22-7.23 S43.99,57.45,43.99,61.44L43.99,61.44L43.99,61.44z M68.15,61.44c0,3.99-3.24,7.22-7.22,7.22c-3.99,0-7.22-3.24-7.22-7.22 c0-3.99,3.24-7.23,7.22-7.23C64.91,54.22,68.15,57.45,68.15,61.44L68.15,61.44L68.15,61.44z M25.2,25.2 C15.93,34.48,10.2,47.29,10.2,61.44c0,14.15,5.74,26.96,15.01,36.24c9.27,9.27,22.08,15.01,36.24,15.01s26.96-5.74,36.23-15.01 c9.27-9.27,15.01-22.08,15.01-36.24c0-14.15-5.74-26.96-15.01-36.23C88.4,15.93,75.59,10.2,61.44,10.2 C47.29,10.2,34.48,15.93,25.2,25.2L25.2,25.2z" />
+                        </g>
+                    </svg>
+                    <button class="explore">
+                        More
+                    </button>
+                </div>
+                <div class="Tweet-button"> <button class="explore-two">
+                        TWEET
+                    </button></div>
+            </div>
+            <div class="profile-menu">
+                <div><img src="/pngegg.png" alt="profile-pic" class="harry"></div>
+                <div class="name-tag">
+                    <div id="display-name">Harry P.</div>
+                    <div id="display-at">@TheBoyWhoLived</div>
+                </div>
+                <div class="more-option-profile">
+                    <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 122.88 29.96">
+                        <defs>
+                            <style>
+                                .cls-1 {
+                                    fill-rule: evenodd;
+                                }
+                            </style>
+                        </defs>
+                        <title>3-horizontal-dots</title>
+                        <path class="cls-1"
+                            d="M0,15A15,15,0,1,1,15,30,15,15,0,0,1,0,15Zm92.93,0a15,15,0,1,1,15,15,15,15,0,0,1-15-15ZM46.46,15a15,15,0,1,1,15,15,15,15,0,0,1-15-15Z" />
+                    </svg>
+                </div>
+            </div>
+        </section>
+
+        <section class="main">
+            <div class="home">
+                <div class="explore-three">Home</div>
+                <div><svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg"
+                        viewBox="0 0 122.88 108.69">
+                        <title>glow</title>
```

Comment:
Interesting, I didn't know that this could be used to show text when hovering over an element